### PR TITLE
Fixed duel module

### PR DESCRIPTION
**Pull request by TroyDota**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

### CODE REVIEW

**Comment by TroyDota** [comment]
The circle is throwing errors on lines i didnt edit?

**Comment by alazymeme** [comment]
Doesn't look like it. Looks like there's been quite a bit of whitespace added to the lines. These issues don't look like they were present before and will need to be fixed.

image
image
image

**Comment by RAnders00** [comment]
Yes, in case you are confused by the output format, it prints a diff where `-` lines should be replaced by `+` lines.

Just run `./scripts/reformat-python.sh` and commit the changes

File: pajbot/modules/duel.py [hunk 1]
```diff
@@ -286,7 +285,7 @@ def accept_duel(self, bot, source, **rest):
 
             # Persist duel statistics
             winner.duel_stats.won(winning_pot)
-            winner.duel_stats.lost(duel_price)
+            loser.duel_stats.lost(duel_price)
```
Comment (RAnders00) [inline pajbot/modules/duel.py]: Good catch. This will definitely have to be fixed asap

File: pajbot/modules/duel.py [hunk 2]
```diff
@@ -236,11 +236,10 @@ def cancel_duel(self, bot, source, **rest):
...
             bot.whisper(source, f"You have cancelled the duel vs {challenged}")
-
-        del self.duel_targets[self.duel_requests[source.id]]
-        del self.duel_requests[source.id]
-        del self.duel_request_price[source.id]
-        del self.duel_begin_time[source.id]
+            del self.duel_targets[challenged.id]
+            del self.duel_request_price[source.id]
+            del self.duel_begin_time[source.id]
+            del self.duel_requests[source.id]
```
Comment (pajlada) [inline pajbot/modules/duel.py]: Please revert the indentation changes made here

**Comment by pajlada** [comment]
Thank you for your contribution!

**Pull request merged**